DOCS-1517 document prometheus prereq (#8383)
DataDog/integrations-core: vault/README.md

#### Prerequisites

Ensure you have enabled [Prometheus metrics in the Vault configuration][16].

> For Vault check to work properly, you need to a) enable unauthenticated access to vault metrics or b) provide a Vault client token.

Reviewer: Should we number these prereqs since there is more than 1.

Author: Yes, good idea ... Getting all the markdown indentation right is the challenge. :) Working on an updated commit for this.

Author: Updated.

#### Prerequisites

1. Ensure you have enabled [Prometheus metrics in the Vault configuration][16].

2. For Vault check to work properly, you need to either enable unauthenticated access to Vault metrics or provide a Vault client token:

   **To enable unauthenticated access**, set Vault [`unauthenticated_metrics_access`][14] configuration to `true`.This will allow unauthenticated access to the `/v1/sys/metrics` endpoint.
   
     **Note**: The `/sys/metrics` endpoint requires Vault v1.1.0 or higher to collect metrics.
   

>     **To use a Vault client token**, follow the example below, which shows using JWT auth method, but you can also use other [auth methods][15]. The capabilities needed for Vault integration to work properly are the following:

Reviewer 2: ```suggestion
    **To use a Vault client token**, follow the example below. The example uses the JWT auth method, but you can also use other [auth methods][15]. 
    
The Vault integration requires the following capabilities:
```
